drivers/scheduler/k8s/k8s.go

```diff
@@ -1099,33 +1119,46 @@ func (k *K8s) createNamespace(app *spec.AppSpec, namespace string, options sched
 
 func convertStorageClassToPure(scParameters map[string]string) map[string]string {
 	if len(scParameters["shared"]) > 0 || len(scParameters["sharedv4"]) > 0 {
-		scParameters["backend"] = PureFile // This is needed to create volumes via Pure FB backend
-		scParameters["exportrules"] = "*(rw)"
-		// These are the parameters that are not supported by Pure FB
-		if _, found := scParameters["shared"]; found {
-			delete(scParameters, "shared")
-		}
-		if _, found := scParameters["sharedv4"]; found {
-			delete(scParameters, "sharedv4")
-		}
-		if _, found := scParameters["secure"]; found {
-			delete(scParameters, "secure")
-		}
-		if _, found := scParameters["repl"]; found {
-			delete(scParameters, "repl")
-		}
-		if _, found := scParameters["scale"]; found {
-			delete(scParameters, "scale")
-		}
-		if _, found := scParameters["aggregation_level"]; found {
-			delete(scParameters, "aggregation_level")
-		}
-		if _, found := scParameters["io_profile"]; found {
-			delete(scParameters, "io_profile")
-		}
-		if _, found := scParameters["priority_io"]; found {
-			delete(scParameters, "priority_io")
-		}
+		return convertStorageClassToPureFB(scParameters)
+	}
+	// TODO: what parameters define Pure FA passthrough volumes?
```

Comment:
TODO, @david-enli or someone else you could you advise me here? (See the above if check for FB, I'm looking for what might be a comparable check for FA).

Reply:
I need to check with Nikolday, @nikolaypopov do you remember why we specifically look for shared / sharedv4 parameter here?